spec/models/user_spec.rb

```diff
@@ -2,35 +2,67 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let(:attributes) do
-    {
-
-      uuid: 'userid:123',
-      email: 'test@test.com',
-      first_name: 'John',
-      middle_name: 'William',
-      last_name: 'Smith',
-      birth_date: Time.new(1980, 1, 1).utc,
-      ssn: '555443333',
-      gender: 'M',
-      level_of_assurance: LOA::THREE
-    }
-  end
+  let(:loa_one) { { current: LOA::ONE } }
+  let(:loa_three) { { current: LOA::THREE } }
 
-  describe 'to create a user' do
+  describe '.create()' do
     context 'with LOA 1' do
+      subject(:loa1_user) { described_class.new(FactoryGirl.build(:user, loa: loa_one)) }
       it 'should allow a blank ssn' do
-        expect(FactoryGirl.build(:user, level_of_assurance: LOA::ONE, ssn: '')).to be_valid
+        expect(FactoryGirl.build(:user, loa: loa_one, ssn: '')).to be_valid
+      end
+      it 'should allow a blank gender' do
+        expect(FactoryGirl.build(:user, loa: loa_one, gender: '')).to be_valid
+      end
+      it 'should allow a blank middle_name' do
+        expect(FactoryGirl.build(:user, loa: loa_one, middle_name: '')).to be_valid
+      end
+      it 'should allow a blank birth_date' do
+        expect(FactoryGirl.build(:user, loa: loa_one, birth_date: '')).to be_valid
+      end
+      it 'should allow a blank zip' do
+        expect(FactoryGirl.build(:user, loa: loa_one, zip: '')).to be_valid
+      end
+      it 'should allow a blank loa.highest' do
+        expect(FactoryGirl.build(:user, loa: { current: LOA::ONE, highest: '' })).to be_valid
+      end
+      it 'should not allow a blank uuid' do
+        loa1_user.uuid = ''
+        expect(loa1_user.valid?).to be_falsey
+        expect(loa1_user.errors[:uuid].size).to be_positive
+      end
+      it 'should not allow a blank email' do
+        loa1_user.email = ''
+        expect(loa1_user.valid?).to be_falsey
+        expect(loa1_user.errors[:email].size).to be_positive
       end
     end
     context 'with LOA 3' do
+      subject(:loa3_user) { described_class.new(FactoryGirl.build(:user, loa: loa_three)) }
       it 'should not allow a blank ssn' do
-        expect(FactoryGirl.build(:user, level_of_assurance: LOA::THREE, ssn: '')).to_not be_valid
+        loa3_user.ssn = ''
+        expect(loa3_user.valid?).to be_falsey
+        expect(loa3_user.errors[:ssn].size).to be_positive
```

Comment:
Tightened up these negative test cases be checking if the specific error was triggered.  Otherwise, they were prone to false positives: the test passing due to an invalid Model but invalid for the wrong reasons.